react/src/components/SessionDetailContent.tsx

```diff
@@ -186,7 +186,7 @@ const SessionDetailContent: React.FC<{
           </Button.Group>
         </Flex>
 
-        <Descriptions bordered column={md ? 2 : 1}>
+        <Descriptions bordered column={md ? 2 : 1} style={{ width: '100%' }}>
```

Comment:
Is there any reason to set width instead of setting `align="stretch"` to parent Flex(line number 160)?

Reply:
I thought Flex in line 160 is for action button so I thought it didn't apply to description even though flex on line 149 had stretch set. I just Fixed.